conf/conf.sh.dist

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+	
+#
+# CUSTOM
+#
+
+DB_WORLD_CUSTOM_PATHS+=(
+    $MOD_WORLD_CHAT_ROOT"/sql/world/"
+)
```

Comment:
always risky to not have a linebreak (i already had the db_assembler fail because of that only on linux ofc)